acceptance/lib/helper.rb

```diff
@@ -61,7 +61,9 @@ def self.get_option_value(value, legal_values, description,
 
     # precedence is environment variable, option file, default value
     value = ((env_var_name && ENV[env_var_name]) || value || default_value)
-    if value
+    if value == ""
+      value = default_value
```

Comment:
For what cases should we be doing a `to_sym` on the value?  Looks like if we get here via the environment variable not being defined and value = "" that what we'll use as the value, `default_value`, won't have `to_sym` applied to it.  If we were to get to using `default_value` because the environment variable was not defined and `value` supplied to the function was nil, though, I think we'd get `default_value.to_sym`.